app/views/registration/register.jade

```diff
@@ -54,8 +54,8 @@ block body
             label(class="text-"+controlGroupClass)
               input(type="checkbox", name="terms", tabindex=tabIndex++)
               |  I confirm that the information I have entered above is not
-              |  fraudulent, and that I agree to be bound by the 
+              |  fraudulent, and that I agree to be bound by the
```

Comment:
Ahh I'm assuming Atom did that on purpose. Must be a setting. I don't remember touching that area. Will fix it, though.